### PR TITLE
README from old upstart-exporter adopted

### PR DESCRIPTION
**Pull request by miros**

### CODE REVIEW

File: init-exporter.go [hunk 1]
```diff
@@ -411,7 +411,7 @@ func showAbout() {
 		App:     APP,
 		Version: VER,
 		Desc:    DESC,
-		Year:    2006,
+		Year:    2016,
```
Comment (andyone) [inline init-exporter.go]: This value must be `2006`, is year when complany was founded.

File: readme.md [hunk 2]
```diff
@@ -13,6 +14,203 @@ make all
...
+often than `count` times in `interval`, it won't be restarted anymore.
+
+Options `working_directory`, `env`, `log`, `respawn` can be
+defined both as global and as per-command options.
+
+#### Exporting
+
+To export a Procfile you should run
+
+    sudo upstart-export -p ./myprocfile -n myapp -f format
```
Comment (andyone) [inline readme.md]: init-exporter not require `-n` argument. Currently command look like `sudo upstart-export -p ./myprocfile -f format myapp`. `-n` still supported only for compatibility with upstart-exporter.

**Comment by andyone** [comment]
@miros Can you add some usage examples to utility? This is [example](https://github.com/essentialkaos/rbinstall/blob/master/cli/cli.go#L813) from rbinstall utility.

**Comment by miros** [comment]
@andyone 

Done!
I have also added make tasks to run upstart/systemd docker images with utility installed to make manual testing easier.

**Comment by andyone** [comment]
@miros Great job! Can you add a new line to the end of each dockerfile to satisfy git?

**Comment by miros** [comment]
@andyone Yes, sure.

**Comment by andyone** [comment]
Thanks! Merged.

**Pull request merged**